monkey/monkey_island/cc/services/remote_run_aws.py

```diff
@@ -51,17 +51,24 @@ def update_aws_auth_params():
         Updates the AWS authentication parameters according to config
```

Comment:
This function isn't used anymore, and it won't be. Don't see a reason not to remove it.
Even if we'll want to use this code from outside an EC2 instance, we can use the credentials stored on the machine in any other way.